api/src/main/java/io/terrakube/api/plugin/scheduler/job/tcl/executor/ephemeral/EphemeralExecutorService.java

```diff
@@ -274,42 +272,20 @@ public ExecutorContext sendToEphemeralExecutor(Job job, ExecutorContext executor
                 .withRestartPolicy("Never")
                 .endSpec()
                 .endTemplate()
+                .withTtlSecondsAfterFinished(30)
                 .endSpec()
                 .build();
 
         log.info("Running ephemeral job");
         try {
-            kubernetesClient.batch().v1().jobs().inNamespace(ephemeralConfiguration.getNamespace()).createOrReplace(k8sJob);
+            kubernetesClient.batch().v1().jobs().inNamespace(ephemeralConfiguration.getNamespace()).resource(k8sJob).serverSideApply();
```

Comment:
The `createOrReplace()` method is deprecated. The new API follows the kubectl style.